bin/node/runtime/src/lib.rs

```diff
@@ -1061,6 +1061,7 @@ impl pallet_vesting::Config for Runtime {
 	type BlockNumberToBalance = ConvertInto;
 	type MinVestedTransfer = MinVestedTransfer;
 	type WeightInfo = pallet_vesting::weights::SubstrateWeight<Runtime>;
+	const MAX_VESTING_SCHEDULES: u32 = 28;
```

Comment:
lets add here a note about the calculation you did about memory size impact

Reply:
@shawntabrizi should we do the same in the polkadot/kusama runtimes?

Reply:
ed9878f